controllers/fenceagentsremediation_controller_test.go

```diff
@@ -133,29 +133,37 @@ var _ = Describe("FAR Controller", func() {
 			It("should have finalizer and taint", func() {
 				farNamespacedName := client.ObjectKey{Name: node01, Namespace: defaultNamespace}
 				nodeNamespacedName := client.ObjectKey{Name: node01}
+				By("Searching for remediation taint")
 				Eventually(func() bool {
 					Expect(k8sClient.Get(context.Background(), nodeNamespacedName, node)).To(Succeed())
```

Comment:
I think that checking for the node existence, which is created in the test code (and not in production code) is redundant.
IMO it's the equivalent of:
```
It("some test", func() {
  someVar := true
  Expect(someVar).To(BeTrue())
})
```

Reply:
On the same principle I can ignore checking the CR creation, right?

Reply:
Even though that I don't agree that I can ignore this line, because without this `k8sClient.Get` **node** variable won't be updated with the node from reconcile with taints that we check on line 141...

Reply:
Good point, definitely need to check the taint on the node.